docs/dev-guide/installingLocal.md

```diff
@@ -128,16 +96,43 @@ The previous shells were launched in their own containers. If you want to open a
   docker exec -it CONTAINER_NAME /bin/bash
   ```
 
-## Using tmux in a container
+<!--
+This content moved to the quickstart.
```

Comment:
What's the plan with this block of comments? I don't understand why this was added.

Reply:
To satisfy the diff algorithm; this will be removed in a follow-up PR.